lexibank_chenhmongmien.py

```diff
@@ -1,15 +1,14 @@
-import csv
+import collections
+from pathlib import Path
 
-import requests
 from bs4 import BeautifulSoup
-from clldutils.misc import slug
-from clldutils.path import Path
 from clldutils.text import strip_brackets, split_text
 from pylexibank.dataset import NonSplittingDataset
 from tqdm import tqdm
-from pylexibank.dataset import Concept, Language
+from pylexibank import Concept, Language
 import attr
 
+
```

Comment:
Important point: the whole `HLanguage` class has far too many duplicates, as the defaults of the normal `pylexibank.models.Language` already contain Latitude, etc. I only later realized that there was no need to add this. So the only thing needed is Chinese Name, SubGroup, Family (due to Default values), Name_in_source, Autonym, and Location.